src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
```

Comment:
its forbidden to use anything except `box-sizing` in `*`. You can reset margin in `body`